### PR TITLE
New registration

### PR DESCRIPTION
**Pull request by dldisha**
Hey maintainer!!

You are doing a great job. Kindly accept my request.

Regards

### CODE REVIEW

File: content/participants/ttjc@dldisha/index.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+type: "participant"
+name: "Disha Lamba"
+portfolio: "https://dldisha.github.io/"
+github-link: "https://github.com/dldisha"
+linkedin-link: "https://www.linkedin.com/in/dldisha/"
+looking-for: "job"
+description: "Full stack Web Developer with Ruby on Rails"
+---
+
+Hi! I am a Disha Lamba, a person with a habit of making things easy and interesting.
```
Comment (khusharth) [inline content/participants/ttjc@dldisha/index.md:12]: Hey! @dldisha i found a small problem here:
* I am a Disha ...

**Comment by khusharth** [comment]
You have done changes in **package-lock.json** file. Please revert that and the projects that you have mentioned are done before jobChallenge. Only projects done during the challenge period are accepted.

**Comment by dldisha** [comment]
@khusharth I have made necessary changes please review it.

**Comment by khusharth** [comment]
@gkanishk Good to merge.

**Comment by gkanishk** [comment]
Welcome, On-board!!!

**Pull request merged**